### PR TITLE
Added an interactive Google Map to the comment page through the Google Maps library

### PR DESCRIPTION
**Pull request by georgezheng999**
Integrated the Google Maps library, which allowed an interactive Google Map to be added.

This map displays the USA underneath the existing comment structure, and users are able to click on spots on the map in order to drop a marker. These markers persist after a session is closed, and are associated with the email address of the user.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -40,6 +40,50 @@ async function deleteComments() {
   });
 }
 
+let map;
+let editMarker;
+
+/** Creates a map that allows users to add markers. */
+function initMap() {
+  const usaLat = 38.5949;
```
Comment (qlyao) [inline portfolio/src/main/webapp/script.js:48]: consider defining these constants at the top of the file instead of inside the function.

**Review by qlyao — APPROVED** [review]

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -58,3 +58,13 @@ h1 {
   display: table;
   clear: both;
 }
+
+#map {
+height: 100%;
```
Comment (ChuxiangP) [inline portfolio/src/main/webapp/style.css]: nit: indent by 2, same below.

**Review by ChuxiangP — APPROVED** [review]
Nice!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (georgezheng999) [reply portfolio/src/main/webapp/script.js:48]: done.

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (georgezheng999) [reply portfolio/src/main/webapp/style.css]: done.

**Pull request merged**